src/renderer/services/chain/fees/unstake.ts

```diff
@@ -0,0 +1,79 @@
+import * as RD from '@devexperts/remote-data-ts'
+import { Chain, BaseAmount } from '@thorchain/asgardex-util'
+import * as FP from 'fp-ts/lib/function'
+import * as O from 'fp-ts/lib/Option'
+import * as Rx from 'rxjs'
+import * as RxOp from 'rxjs/operators'
+
+import { BASE_CHAIN } from '../../../const'
+import { isBaseChain } from '../../../helpers/chainHelper'
+import { liveData, LiveData } from '../../../helpers/rx/liveData'
+import { observableState } from '../../../helpers/stateHelper'
+import * as BNB from '../../binance/service'
+import * as BTC from '../../bitcoin/context'
+import { selectedPoolChain$ } from '../../midgard/common'
+import { getCrossChainUnstakeMemo$ } from '../memo'
+import { FeeLD, StakeFeesLD } from '../types'
+import { reloadStakeFeesByChain } from './fees.helper'
+
+const { get$: unstakePercent$, set: updateUnstakePercent } = observableState(0)
```

Comment:
How about to add `get: unstakePercent` , so we can get latest value of it without a need to subscribe. It will reduce subscription for example in `unstakeFeeByChain$`